### PR TITLE
Some fixes and cod improvements

### PR DESCRIPTION
**Pull request by ValentinNikolaev**
- move Zabbix error to Exception; 
- PSR-1/2 code style; 
- fix ZabbixPrimitiveItem issue; 
- markup changes

### CODE REVIEW

File: src/ZabbixAgent.php [hunk 1]
```diff
@@ -3,48 +3,40 @@
...
-class ZabbixAgent {
+class ZabbixAgent
+{
     /**
      * Items on this agent
-     * @var array 
+     * @var array
      */
-    protected $items;
+    protected $items = [];
```
Comment (wizardjedi) [inline src/ZabbixAgent.php]: I think It's better ti use array() instead of [] because of version of php.

File: src/ZabbixAgent.php [hunk 2]
```diff
@@ -3,48 +3,40 @@
...
-    /**
-     * Create zabbix agent object
      * @param string $host
      * @param int $port
      * @throws ZabbixAgentException
      */
-    function __construct($host, $port) {
+    function __construct($host, $port)
+    {
         if (empty($host)) {
```
Comment (wizardjedi) [inline src/ZabbixAgent.php:40]: Maybe check port too?

File: src/ZabbixNotSupportedItem.php [hunk 3]
```diff
@@ -1,20 +1,24 @@
...
-    public function create($message) {
-        return new ZabbixNotSupportedItem($message);
+    function __construct($message)
+    {
+        $this->message = $message;
     }
 
-    function __construct($message) {
-        $this->message = $message;
+    public function create($message)
```
Comment (wizardjedi) [inline src/ZabbixNotSupportedItem.php]: Must be public static function.

**Review by wizardjedi — CHANGES_REQUESTED** [review]

File: src/ZabbixAgent.php (hunk 1 above)
Comment (ValentinNikolaev) [reply src/ZabbixAgent.php]: < PHP 5.4 ?

File: src/ZabbixAgent.php (hunk 2 above)
Comment (ValentinNikolaev) [reply src/ZabbixAgent.php:40]: Nothig changed there. It still checks for port.

File: src/ZabbixAgent.php (hunk 1 above)
Comment (wizardjedi) [reply src/ZabbixAgent.php]: Yes :)

Comment (wizardjedi) [reply src/ZabbixAgent.php]: Only one issue last for merge ;)

Comment (ValentinNikolaev) [reply src/ZabbixAgent.php]: I'll fix in an hour :+1:

**Review by wizardjedi — APPROVED** [review]

**Pull request merged**